Client/Assets/Search/SearchPlugins/en/yahoo.xml

```diff
@@ -5,18 +5,25 @@
 <Url type="application/x-suggestions+json" method="GET" template="https://search.yahoo.com/sugg/ff">
-    <Param name="output" value="fxjson"/>
-    <Param name="appid" value="ffm"/>
-    <Param name="command" value="{searchTerms}"/>
-    <Param name="nresults" value="8"/>
+  <Param name="output" value="fxjson"/>
+  <Param name="appid" value="ffm"/>
+  <Param name="command" value="{searchTerms}"/>
+  <Param name="nresults" value="4"/>
```

Comment:
I think this change is fine -- it corresponds to the number of results shown in suggestions. We limit that number to 4 anyway in the UI, so this should have no effect. Also worth noting that Android used to have `nresults` set to 8 when we originally pulled these in, so this was not an iOS-specific change.